src/registries/data_validation_registry.ts

```diff
@@ -565,6 +589,7 @@ dataValidationEvaluatorRegistry.add("isBetween", {
     return _t("The value must be between %s and %s", values[0], values[1]);
   },
   isCriterionValueValid: (value) => checkValueIsNumber(value),
+  isCriterionFormulaValueValid: (value) => isValidFormula(value),
```

Comment:
I don't understand the point of this registry key. We want to check that the formula is valid every time we have a formula no, we don't need a specific registry key for this ? Why do we sometime want to just return true without checking the validity of the formula ?